src/main/java/io/confluent/connect/elasticsearch/DataConverter.java

```diff
@@ -78,10 +83,18 @@ private static String convertKey(Schema keySchema, Object key) {
     }
   }
 
-  public static IndexableRecord convertRecord(SinkRecord record, String index, String type, boolean ignoreKey, boolean ignoreSchema) {
+  public static IndexableRecord convertRecord(
+      SinkRecord record,
+      String index,
+      String type,
+      boolean ignoreKey,
+      boolean ignoreSchema
+  ) {
     final String id;
     if (ignoreKey) {
-      id = record.topic() + "+" + String.valueOf((int) record.kafkaPartition()) + "+" + String.valueOf(record.kafkaOffset());
+      id = record.topic()
+           + "+" + String.valueOf((int) record.kafkaPartition())
```

Comment:
Same question as below with the alignment of literals in multiple lines (see comment on ternary operator alignment)